### PR TITLE
sig-cluster-lifecycle: Update meeting times

### PR DESCRIPTION
**Pull request by randomvariable**
Updating SIG Cluster Lifecycle meetings according to prior Zoom calls and emails.

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -688,22 +688,30 @@ sigs:
...
-      tz: "PT (Pacific Time)"
-      frequency: weekly
+    - description: Cluster API Provider Implementers' office hours (EMEA)
+      day: Tuesday
+      time: "17:00"
+      tz: "London"
+      frequency: biweekly
       url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
-      archive_url: https://docs.google.com/document/d/10dq54Fd-xa6P5Iy3p46VY1YTFqugGMd1PygDIpuRw6c/edit
+      archive_url: https://docs.google.com/document/d/16ils69KImmE94RlmzjWDrkmFZysgB2J4lGnYMRN89WM/edit#
```
Comment (roberthbailey) [inline sigs.yaml]: This notes URL is for the primary cluster api meeting. Does the emea office hours have a separate notes doc?

Comment (roberthbailey) [reply sigs.yaml]: Looks like the doc for US West Coast (https://docs.google.com/document/d/1IZ2-AZhe4r3CYiJuttyciS7bGZTTx4iMppcA8_Pr3xE/edit) was meant to be shared with the emea meeting as well. Please update this link.

Comment (randomvariable) [reply sigs.yaml]: Doh. Thanks.

**Pull request merged**